scripts/build.py

```diff
@@ -141,6 +165,8 @@ def get_cache_slug():
     CACHE_SLUG = get_cache_slug()
     BUILD_DIR = os.path.join('build', CACHE_SLUG)
 
+    # ensure_directory_exists trims file paths passed to it. Hence, directory
+    # paths require a trailing slash.
```

Comment:
I disagree; the comment is confusing and needs to be rewritten. I don't see the connection between trimming a file path (which I tend to understand as removing leading and trailing whitespace) and requiring a trailing slash for directory paths. These seem like two distinct concepts that are somehow being forced to relate to each other in these comments, but it's not working. I spent about 3 minutes trying to understand what these comments meant, looked around at different functions and I still don't understand it. Comments should definitely explain something easier than code does, and neither code nor the comments are explaining the concept to me.

I know this was copied from above, but please rewrite it to be more clear. Thanks!